custom_components/artnet_led/client/__init__.py

```diff
@@ -373,6 +375,8 @@ class TimeCodeType(Enum):
 
 
 class ArtBase:
+    __ENCODINGS__ = ['ascii', 'utf8']
```

Comment:
Remove `ascii`, add `iso-8859-1` after `utf8`, that seems to be the most popular ASCII extension and shouldn't be able to fail.